shell/platform/tizen/channels/platform_view_channel.cc

```diff
@@ -191,17 +213,12 @@ void PlatformViewChannel::HandleMethodCall(
       } else if (method == "setDirection") {
         FT_LOGD("PlatformViewChannel setDirection");
         result->NotImplemented();
-      } else if (method == "clearFocus") {
```

Comment:
Why remove this :question:

Reply:
I did not remove it and moved just the location of the code.

Reply:
Oh.. boy.  Sorry! :sweat_smile: